src/renderer.cpp

```diff
@@ -1228,6 +1506,11 @@ uint32_t Renderer::blitTexture(uint32_t x, uint32_t y, uint32_t width, uint32_t
     return ret.idx;
 };
 
+void Renderer::setDrawType(DrawType drawType)

```

Comment:
If this function is about setting the Light draw type, it should be named accordingly `setLightDrawType` no?

Reply:
I intentionally set the name somehow ambiguous because this may be used for other purposes than lighting in the future.

Reply:
I do prefer having it named accordingly when the body of the function manipulates only light draw types. When and if that will happen, I do prefer to see a commit changing the name of this method, instead of reserving it :) Thanks

Reply:
I renamed it to LightingDrawType.